README.md

```diff
@@ -39,7 +39,7 @@ Command-line options can be grouped into these categories:
                              with -f
     -m str, --method str     HTTP method to use. Default is 'GET'
     -a str, --agent str      Set User-Agent request header
-    -H str, --header str     Send request header in 'name:value' format. Specify
+    -H str, --headers str    Send request headers in 'name:value' format. Specify
```

Comment:
I believe it makes more sense as `--header` since each invocation of `--header` specifies only one header. Sending multiple headers looks like:
```
mobbage http://localhost:8080 --header foo:bar --header baz:qux
```

Alternatively:
```
mobbage http://localhost:8080 -H foo:bar -H baz:qux
```

Or some combination of those.

If you feel strongly that it should be `--headers`, though, I'd like to understand your reasoning.